kinto_emailer/tests/test_includeme.py

```diff
@@ -210,6 +210,28 @@ def test_get_messages_can_filter_by_event_class(self):
         assert len(messages) == 1
 
 
+class GroupExpansionTest(unittest.TestCase):
+
+    def test_recipients_are_expanded_from_group_members(self):
+        storage = mock.MagicMock()
+        collection_record = {
+            'kinto-emailer': {
+                'hooks': [{
+                    'template': 'Poll changed.',
+                    'recipients': ['/buckets/b/groups/g'],
```

Comment:
Let's do that in another issue : validation of hooks. I spawned https://github.com/Kinto/kinto-emailer/issues/21